tests/pytests/test_profile.py

```diff
@@ -535,7 +535,7 @@ def TimedOutWarningtestCoord(env):
   env.expect('FT.CREATE', 'idx', 'SCHEMA', 't', 'TEXT').ok()
 
   # Populate the index
-  num_docs = 20000 * env.shardsCount
+  num_docs = 30000 * env.shardsCount
```

Comment:
```suggestion
  num_docs = 30000 * env.shardsCount if not SANITIZER else 3000 * env.shardsCount
```

Reply:
This might result in flakiness, I'd rather the test take more time and be non-flaky, but anyways there's no reason to run this with sanitizers since it's not a unique flow